### ISSUE TITLE
Investigate extremely slow requests captures on Edge environment

### ISSUE THREAD

**Issue by alex-fedotyev** [elastic/opbeans-node#99]
Observing the app performance overtime on Edge environment, I noticed that it sometimes has rare but extremely slow transactions (2-5 minutes slow).
![image](https://user-images.githubusercontent.com/61838744/131566156-fd31da89-8311-44c3-a186-c5a262e5f14d.png)

Details don't capture why this could be happening.
![image](https://user-images.githubusercontent.com/61838744/131566258-86ef5643-d614-405e-922a-e5ee786dd777.png)

I suggest to investigate a little to better understand what could be happening.
cc @trentm

**Comment by trentm** [comment]
Thanks. I poked a little bit -- mostly me learning some about opbeans-node and opbeans-loadgen -- and I don't see any obvious normal reason why the "GET /api/customers" endpoint code could have a long delay at the end: https://github.com/elastic/opbeans-node/blob/a052ed72fed409fa3eb8a9300a146ea6a9b54d3b/server/routes.js#L167-L180

- There is the separate handler to redirect "/api" requests 50% of the time that I'm not at all familiar with: https://github.com/elastic/opbeans-node/blob/a052ed72fed409fa3eb8a9300a146ea6a9b54d3b/server.js#L86-L109
- and the `app.get('*', ...` routing pattern that I'm not familiar with: https://github.com/elastic/opbeans-node/blob/a052ed72fed409fa3eb8a9300a146ea6a9b54d3b/server.js#L113
- I noticed that some of the other `opbeans-$lang` services also have some slow outliers as well.
- The slow transactions on edge *today* are all "GET /api/customers".

tl;dr: I don't have much useful info.  @alex-fedotyev If something changes to either make this a higher prio or if you notice a behaviour change, please mention it here.
